x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/vectors/50_dense_vector_field_usage.yml

```diff
@@ -0,0 +1,87 @@
+setup:
```

Comment:
This test might need a skip section for <8.1 for mixed cluster tests.

Reply:
hmm, don't really understand why this passes test. Maybe we don't run these in x-pack on mixed clusters?

Reply:
yes, this validates my understanding (I deliberately did not add the version constraint to check if anything breaks on CI). I find the findings very worrying though.

Reply:
Just see this passed, so nevermind. I don't really know why this works in mixed cluster tests wasn't aware that yml in x-pack isn't run against

Reply:
Good to see that we both expected this to fail in some way. Maybe worth some parallel investigation, I'd be interested in the results.

Reply:
It seems we don't have "mixed cluster" tests for x-pack like we do for the core apis. So we simply aren't running this test under that scenario at all.

https://github.com/elastic/elasticsearch/issues/31096

Reply:
Wow, that's scary (and lol for linking to my own issue, guess my memory is failing me).

Perhaps that one needs to be reprioritized!

Reply:
The issue seems to be tagged as core/infra but I'm not sure if that's strictly correct. There's no real distinct ownership of these large cross-functional test suites. It's really just a matter of adding a qa project for this under x-pack similar to the existing core one.